Automattic/newspack-plugin
# Ensure a single instance of react-router-dom is used

### All Submissions:






### Changes proposed in this Pull Request:



`assets/components` is a small npm package on it's own, and as such it may contain a `node_modules` folder. In that case, if a node module was imported by a component in `assets/components` and one in, say, `assets/wizards`, they would end up with separate copies of the module. This caused issues with `react-router-dom`.
This was resolved by proxying `react-router-dom` as a module in `assets/components` - a single source of truth for this external module. 

### How to test the changes in this Pull Request:

1. run `npm i` in `assets/components` to create `node_modules` there
2. build the plugin JS (`npm run build`) and navigate to a wizard (e.g. Analytics)
3. everything should work as expected

### Other information:
